integration/pod_userns_linux_test.go

```diff
@@ -143,12 +143,17 @@ func TestPodUserNS(t *testing.T) {
 		t.Fatalf("failed to setup volume host path: %v", err)
 	}
 
+	hostname, err := os.Hostname()
+	require.NoError(t, err)
+
 	for name, test := range map[string]struct {
-		sandboxOpts   []PodSandboxOpts
-		containerOpts []ContainerOpts
-		checkOutput   func(t *testing.T, output string)
-		hostVolumes   bool // whether to config uses host Volumes
-		expectErr     bool
+		sandboxOpts      []PodSandboxOpts
+		containerOpts    []ContainerOpts
+		checkOutput      func(t *testing.T, output string, expectedHostn string)
+		hostVolumes      bool // whether to config uses host Volumes
+		expectErr        bool
+		expectedHostname string
```

Comment:
A hostname is the concept of UTSNS, not NetNS.

Does `hostNetwork: true` imply using the host UTS namespace too?